controllers/teams.js

```diff
@@ -1044,6 +1052,7 @@ router.get('/:teamId/members', async (req, res, next) => {
 				{
 					payload: {
 						userId: user.userId._id,
+						role: userRole && userRole.name,
```

Comment:
Why not use a boolean flag here (á la "could be team admin"), so you can remove the "role===student"-check later?